src/components/NewWorkspaceModal.js

```diff
@@ -249,6 +249,14 @@ const NewWorkspaceModal = withDisplayName('NewWorkspaceModal', ({
           options: _.sortBy('label', isAlphaRegionalityUser ? allRegions : availableBucketRegions)
         })
       ])]),
+      isLocationMultiRegion(bucketLocation) && div({ style: { ...warningStyle } }, [
+        icon('warning-standard', { size: 24, style: { color: colors.warning(), flex: 'none', marginRight: '0.5rem' } }),
+        div({ style: { flex: 1 } }, [
+          `Effective October 1, 2022, Google Cloud will charge egress fees on data stored in multi-region storage buckets.`,
+          p(`Choosing a multi-region bucket location may result in additional storage costs for your workspace.`),
+          p(`Unless you require geo-redundancy for maximum availabity for your data, you should choose a single region bucket location. For more information see <LINK TO BLOG POST>.`)
```

Comment:
TODO: I need to get a real link in this copy

Reply:
Maybe https://support.terra.bio/hc/en-us/articles/360060777272-US-regional-versus-Multi-regional-US-buckets-trade-offs#h_01F5GJ8B50DDV28N92P2BPN12P ?

Reply:
https://support.terra.bio/hc/en-us/articles/360060777272-US-regional-versus-Multi-regional-US-buckets-trade-offs Does seem like the right article, but someone should follow up with Allie to get it corrected to match this change. For instance, it currently says "If you don't need your data to be stored in multiple US regions, you can save storage costs in Terra by changing from the default (multi-regional) to regional cloud storage buckets.", and it doesn't mention Montreal.

Reply:
I have added the needed link to a forthcoming new article. I'll make sure that it's live for this change.